app/views/directives/logs/_log-viewer.html

```diff
@@ -53,16 +47,8 @@ <h2>Logs are not available.</h2>
     {{emptyStateMessage}}
   </p>
 
-  <div ng-if="kibanaAuthUrl">
-    <form
-      action="{{kibanaAuthUrl}}"
-      method="POST">
-      <input type="hidden" name="redirect" value="{{kibanaArchiveUrl}}">
-      <input type="hidden" name="access_token" value="{{access_token}}">
-      <button class="btn btn-primary btn-lg">
-        View Archive
-      </button>
-    </form>
+  <div ng-if="kibanaArchiveUrl">
```

Comment:
@spadgett why is this form here again?

Reply:
It was there to POST the user's API token to kibana to avoid another login